custom/icds_reports/ucr/data_sources/ccs_record_cases_monthly_tableau2.json

```diff
@@ -1760,6 +1760,23 @@
           "name": "postnatal"
         },
         "type": "boolean"
+      },
+      {
+        "column_id": "add",
+        "datatype": "date",
+        "display_name": null,
+        "expression": {
+          "expression": {
+            "datatype": null,
+            "property_name": "add",
+            "type": "property_name"
+          },
+          "type": "root_doc"
+        },
+        "is_nullable": false,
```

Comment:
Actually, shouldn't this be nullable = true?

Reply:
Oh. wow. yes. thanks. Fixed this